.changeset/brown-news-nail.md

```diff
@@ -0,0 +1,5 @@
+---
+'skeleton': patch
+---
+
+Refactor pagination previous & next behaviour into a re-usable component.
```

Comment:
I am ok with deleting as well. I mostly did this to ensure a skeleton & create-hydrogen update is release.
Maybe we can chat about how this can be done without adding something to the release note?